jenkins/jenkins2-ha-agents.yaml

```diff
@@ -1151,23 +1151,23 @@ Resources:
           files:
             '/root/plugins.txt':
               content: |
-                https://updates.jenkins-ci.org/download/plugins/jqs-monitoring/1.4/jqs-monitoring.hpi
-                https://updates.jenkins-ci.org/download/plugins/ant/1.11/ant.hpi
-                https://updates.jenkins-ci.org/download/plugins/build-timeout/1.20/build-timeout.hpi
-                https://updates.jenkins-ci.org/download/plugins/docker-workflow/1.23/docker-workflow.hpi
-                https://updates.jenkins-ci.org/download/plugins/credentials-binding/1.23/credentials-binding.hpi
-                https://updates.jenkins-ci.org/download/plugins/email-ext/2.73/email-ext.hpi
-                https://updates.jenkins-ci.org/download/plugins/envinject/2.3.0/envinject.hpi
-                https://updates.jenkins-ci.org/download/plugins/github-organization-folder/1.6/github-organization-folder.hpi
-                https://updates.jenkins-ci.org/download/plugins/gradle/1.36/gradle.hpi
-                https://updates.jenkins-ci.org/download/plugins/workflow-aggregator/2.6/workflow-aggregator.hpi
-                https://updates.jenkins-ci.org/download/plugins/pipeline-maven/3.8.2/pipeline-maven.hpi
-                https://updates.jenkins-ci.org/download/plugins/ssh-slaves/1.31.2/ssh-slaves.hpi
-                https://updates.jenkins-ci.org/download/plugins/subversion/2.13.1/subversion.hpi
-                https://updates.jenkins-ci.org/download/plugins/timestamper/1.11.5/timestamper.hpi
-                https://updates.jenkins-ci.org/download/plugins/ws-cleanup/0.38/ws-cleanup.hpi
-                https://updates.jenkins-ci.org/download/plugins/pipeline-aws/1.42/pipeline-aws.hpi
-                https://updates.jenkins-ci.org/download/plugins/http_request/1.8.26/http_request.hpi
+                http://updates.jenkins-ci.org/download/plugins/jqs-monitoring/1.4/jqs-monitoring.hpi
```

Comment:
Why did you switch from `https` to `http`? How does Jenkins ensure the integrity of the plugins?

Reply:
https is not working :)